opentelemetry-api/src/opentelemetry/util/types.py

```diff
@@ -56,3 +56,18 @@
     ],
     ...,
 ]
+
+
+@dataclass
+class MetricsHistogramAdvisory:
+    explicit_bucket_boundaries: Optional[Sequence[float]] = None
+
+
+@dataclass
+class MetricsCommonAdvisory:
+    pass
+
+
+MetricsInstrumentAdvisory = Union[
+    MetricsCommonAdvisory, MetricsHistogramAdvisory
+]
```

Comment:
Not 100% clear what this is for, but would using dataclass inheritance here work? https://docs.python.org/3/library/dataclasses.html#inheritance

Reply:
This is to make explicit that we have two types of advisory, the common one that will get `attributes` and the histogram specific one.